### ISSUE TITLE
Repo is big

### ISSUE THREAD

**Issue by jmcarp** [caneruguz/treebeard#59]
This is very minor, but this repo is really big (about 200MB checked out locally), mostly because of some large json files in the demo folder. Can those files be removed? It looks like they can be recreated by `generate.js`, so we shouldn't have to include them in the repo.

**Comment by caneruguz** [comment]
Good point. Steve and I were talking about this yesterday. As a result I made this change:
https://github.com/caneruguz/treebeard/commit/89a6e717d26253b2783df8d34530b750f3377de3

Let me know if other changes can be made. 

For installations @sloria added the ignore section so the people installing through bower should not have any of the demo and unrelated files: https://github.com/caneruguz/treebeard/blob/develop/bower.json
